src/main/groovy/org/transmartproject/batch/db/PostgresPartitionTasklet.groovy

```diff
@@ -132,6 +147,7 @@ class PostgresPartitionTasklet implements Tasklet {
             partitionId = matcher[0] as int
         }
 
+        //FIXME Where we delete data if table already exists
```

Comment:
Old records are deleted with a `DELETE FROM <main table>`. This will affect the partitions as well.